wireguard-install.sh

```diff
@@ -115,6 +115,14 @@ function installQuestions() {
 			CLIENT_DNS_2="${CLIENT_DNS_1}"
 		fi
 	done
+	
+	#Allowed IPs
+	until [[ ${ALLOWED_IPS} =~ ^.+$ ]]; do
+		read -rp "Allowed IPs list: " -e -i '0.0.0.0/0, ::/0, 128.0.0.0/1' ALLOWED_IPS
```

Comment:
What's the purpose of `128.0.0.0/1`?

Reply:
https://www.wireguard.com/netns/

```
The Classic Solutions
The classic solutions rely on different types of routing table configurations. For all of these, we need to set some explicit route for the actual WireGuard endpoint. For these examples, let's assume the WireGuard endpoint is demo.wireguard.com, which, as of writing, resolves to 163.172.161.0. Further, let's assume we usually connect to the Internet using eth0 and the classic gateway of 192.168.1.1.

Replacing The Default Route
The most straightforward technique is to just replace the default route, but add an explicit rule for the WireGuard endpoint:

# ip route del default
# ip route add default dev wg0
# ip route add 163.172.161.0/32 via 192.168.1.1 dev eth0
This works and is relatively straightforward, but DHCP daemons and such like to undo what we've just did, unfortunately.

Overriding The Default Route
So, instead of replacing the default route, we can just override it with two more specific rules that add up in sum to the default, but match before the default:

# ip route add 0.0.0.0/1 dev wg0
# ip route add 128.0.0.0/1 dev wg0
# ip route add 163.172.161.0/32 via 192.168.1.1 dev eth0
This way, we don't clobber the default route. This also works quite well, though, unfortunately when eth0 goes up and down, the explicit route for demo.wireguard.com will be forgotten, which is annoying.
```

Reply:
See https://serverfault.com/a/1100264, so:


```suggestion
		read -rp "Allowed IPs list: " -e -i '0.0.0.0/1, 128.0.0.0/1, ::/0,' ALLOWED_IPS
```

Reply:
Thanks.